### PR TITLE
IGAPP-1137: Maps attribution

### PR DESCRIPTION
**Pull request by f1sh1918**
OpenStreetMaps attribution will now be displayed for web and native

Requirements are fulfilled (https://www.openstreetmap.org/copyright)
- show correct text: © OpenStreetMap contributors (https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines)
- provide link to: https://www.openstreetmap.org/copyright

Note: To show the attribution icon for mobile web, i had to reposition the geocontrol button. I did it same way as on native to reposition if bottom action sheet changes. Additionally i hided the unneeded `NavigationControl` on mobile since you zoom with your fingers

Additionally i deleted `MapPopup` since its not used since long time

### CODE REVIEW

File: native/src/components/Link.tsx [hunk 1]
```diff
@@ -14,13 +15,14 @@ const LinkText = styled.Text`
 type LinkProps = {
   url: string
   text: string
+  style?: StyleProp<TextStyle>
```
Comment (f1sh1918) [inline native/src/components/Link.tsx:18]: needed to override standard padding

**Review by LeandraH — APPROVED** [review]
Code looks good, only tested in Firefox, my native build is currently not working

File: web/src/components/MapAttribution.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { openStreeMapCopyright } from 'api-client'
+
+import CleanLink from './CleanLink'
+
+const Attribution = styled.div`
+  display: flex;
+  padding: 0 4px;
+  background-color: rgba(255, 255, 255, 0.3);
+  box-shadow: 0 2px 3px 3px rgb(0 0 0 / 10%);
+  color: rgba(0, 0, 0, 0.75);
```
Comment (sarahsporck) [inline web/src/components/MapAttribution.tsx:13]: Should be the same as in the footer overlay imo.
```suggestion
  background-color: rgba(255, 255, 255, 0.5);
  box-shadow: 0 2px 3px 3px rgba(0, 0, 0, 0.1);
  color: rgba(0, 0, 0, 0.75);
```

**Review by sarahsporck — CHANGES_REQUESTED** [review]
Copyright not visible in web mobile. (seems to be below BottomActionSheet)
Also link seems to be missing in web desktop.
(Tested on Safari)

Native is fine :)

**Comment by f1sh1918** [comment]
> Copyright not visible in web mobile. (seems to be below BottomActionSheet) Also link seems to be missing in web desktop. (Tested on Safari)
> 
> Native is fine :)

@sarahsporck
What do you mean by link seems to be missing?
Works fine for me on safari desktop

I also tested the geocontrol on safari mobile view and its slow but working. I guess this is due to the emulator and gonna test it on my iPhone

https://user-images.githubusercontent.com/37902063/220652554-cc2e6378-3ae8-4ecb-ba81-1d5896dc2b7e.mov

File: native/src/components/MapsAttribution.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+`
+
+const Label = styled.Text`
+  padding: 4px;
+`
+
+const Copyright = styled.Text<{ expanded: boolean }>`
+  padding-right: 4px;
+  color: rgba(0, 0, 0, 0.75);
+  font-size: ${props => (props.expanded ? '14px' : '20px')};
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply native/src/components/MapsAttribution.tsx:41]: these fonts are in rem and can't be used for native `decorativeFontSizeSmall: '0.7rem'`

[the remark this answers is not in the document]
Comment (sarahsporck) [reply native/src/components/MapsAttribution.tsx:41]: ah okay. 👍🏼

**Review by sarahsporck — APPROVED** [review]
Tested in Safari again and works fine now. Maybe I was on the wrong branch while testing web. ^^

**Pull request merged**